pip/commands/install.py

```diff
@@ -251,12 +251,12 @@ def run(self, options, args):
                     progress_bar=options.progress_bar,
                 )
 
-                self.populate_requirement_set(
-                    requirement_set, args, options, finder, session, self.name,
-                    wheel_cache
-                )
-
                 try:
+                    self.populate_requirement_set(
```

Comment:
Seems like it's needed - https://travis-ci.org/pradyunsg/pip/jobs/249302943.

This isn't a blocker then for this PR - if someone would be able to explain to me why this is needed though it'd be nice.